lib/lotus/lotusrc.rb

```diff
@@ -132,6 +113,16 @@ def exists?
       path_file.exist?
     end
 
+    private
+
+    def symbolize(hash)
+      Utils::Hash.new(hash).symbolize!
+    end
+
+    def file_options
+      symbolize(exists? ? Dotenv::Parser.call(content) : {})
```

Comment:
@aderyabin We assume that this now exists for all Lotus apps, no need for this check here. That would make `#exists?` unnecessary at this point.